cmd/utils/dev.go

```diff
@@ -68,6 +68,14 @@ func LoadManifest(devPath string) (*model.Manifest, error) {
 		return nil, err
 	}
 
+	if manifest.Name == "" {
```

Comment:
This potentially could affect to other places where `LoadManifest` was being used. I guess it is ok, but mentioning it just in case.